File: crates/enforcer-memory/tests/fixtures/memory/lang_swift/widget.swift
import Foundation

protocol Drawable {
    func draw() -> String
}

class Widget: Drawable {
    let name: String

    init(name: String) {
        self.name = name
    }

    func draw() -> String {
        if name.isEmpty {
            return "unnamed"
        }
        return helper(name)
    }
}

func helper(_ label: String) -> String {
    return label.uppercased()
}

File: crates/enforcer-memory/tests/fixtures/memory/lang_kotlin/widget.kt
package widget

import kotlin.math.max

interface Drawable {
    fun draw(): String
}

class Widget(val name: String) : Drawable {
    override fun draw(): String {
        if (name.isEmpty()) {
            return "unnamed"
        }
        return helper(name)
    }
}

fun helper(label: String): String {
    return label.uppercase()
}
